public/apps/configuration/utils/test/password-edit-panel.test.tsx

```diff
@@ -24,18 +39,47 @@ describe('Password edit panel', () => {
   const updateIsInvalid = jest.fn();
   const useState = jest.spyOn(React, 'useState');
   const useEffect = jest.spyOn(React, 'useEffect');
+  const mockCoreStart = {
+    http: 1,
+  };
 
   beforeEach(() => {
     useEffect.mockImplementationOnce((f) => f());
     useState.mockImplementation((initialValue) => [initialValue, setState]);
+    (getDashboardsInfo as jest.Mock).mockImplementation(() => {
```

Comment:
Is this still need even though it is mocked above?

Reply:
It is not, I removed this and also removed creating the shallow component in beforeEach and moved it into each individual test. The first test needs `mount(...)` instead of `shallow(...)` to test the logic in `React.useEffect(...)`